### PR TITLE
chore: add missing deps

### PR DESCRIPTION
**Pull request by lifeiscontent**
Fixes failure from CI during release: https://github.com/edmundhung/conform/actions/runs/9275930573/job/25521773327

### CODE REVIEW

File: packages/conform-dom/.gitignore [hunk 1]
```diff
@@ -1,7 +1,8 @@
 _virtual
+!rollup.config.js
 *.cjs
 *.d.ts
 *.js
 *.mjs
+LICENSE
```
Comment (edmundhung) [inline packages/conform-dom/.gitignore:7]: We are not copying the LICENSE anymore as tpnpm will handle that for us

File: packages/conform-dom/.gitignore [hunk 2]
```diff
@@ -1,7 +1,8 @@
 _virtual
+!rollup.config.js
```
Comment (edmundhung) [inline packages/conform-dom/.gitignore:2]: Is there any difference with the order?

**Review by edmundhung — COMMENTED** [review]
Hey! Sorry I didn't notice you made this PR and I have just merged another PR that add the missing pkg.

File: packages/conform-dom/.gitignore (hunk 2 above)
Comment (lifeiscontent) [reply packages/conform-dom/.gitignore:2]: no, I just sorted it for readability

File: packages/conform-dom/.gitignore (hunk 1 above)
Comment (lifeiscontent) [reply packages/conform-dom/.gitignore:7]: when I ran `pnpm i` I noticed the prepare script copy it into the folder

**Comment by lifeiscontent** [comment]
@edmundhung if this is all sorted in `main` feel free to close, was just trying to help :)

File: packages/conform-dom/.gitignore (hunk 1 above)
Comment (edmundhung) [reply packages/conform-dom/.gitignore:7]: You might be running an older version. I removed the [settings for copying the LICENSE last night](https://github.com/edmundhung/conform/pull/649/commits/97fe6f6083461e43174e9fe5befa8013865f915e).

**Comment by edmundhung** [comment]
Closing this as the missing package is already added. Thanks again :)

**Pull request closed without merging**